index.toml

```
@@ -145,4 +145,12 @@ description = "Use a Retriever and a query generator to perform unsupervised dom
 level = "advanced"
 weight = 140
 notebook = "18_GPL.ipynb"
-aliases = ["gpl"]
\ No newline at end of file
+aliases = ["gpl"]
+
+[[tutorial]]
+title = "Text-to-Image search pipeline with MultiModal Retriever"
```

```suggestion
title = "Text-To-Image Search Pipeline with Multimodal Retriever"
```
we use Chicago style

---

Thanks :) Fixed.